debian/bionic/foreman/rules

```diff
@@ -11,6 +11,7 @@ build:
 	/usr/bin/gem install -f rake -v '12.3.1'
 	/usr/bin/gem install -f minitest -v '5.10.3'
 	/usr/bin/gem install -f thor -v '0.19.4'
+	/bin/rm -f bundler.d/test.rb bundler.d/development.rb
```

Comment:
Nit: I'd drop the `-f` since that'll tell us when we (for some reason) not have these files which would be highly unexpected.